docs/examples/SQLiteExample/README.md

```diff
@@ -0,0 +1,18 @@
+title: SQLite Example
+description: A Loom Example!
+source: src/SQLiteExample.ls
+thumbnail: images/screenshot.png
+!------
+
+## Overview
+
+This example demonstrates SQLite
```

Comment:
Can we clarify what it does to demonstrate Sqlite? Also, is the screenshot present in this pull?